frontend/src/data/fetchWithToken.ts

```diff
@@ -42,6 +42,74 @@ export async function fetchWithToken<T>(path: string): Promise<T | undefined> {
   }
 }
 
+export async function putWithToken<T>(path: string): Promise<T | undefined> {
```

Comment:
Denne filen kunne kanskje blitt delt opp eller blitt navngitt noe annet? Siden den nå også inneholder putWithToken og postWithToken

Reply:
Kan sikkert dele den opp 😄

Reply:
Eller, kanskje heller skrive den om, for nå er alle egnetlig helt identiske bortsett fra forskjellige options